src/StorageCollection.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+/*
+ * This file is part of the PHP Translation package.
+ *
+ * (c) PHP Translation team <tobias.nyholm@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Translation\Common;
```

Comment:
Should it be moved into a `Storage` subfolder?

Reply:
Yes, I think it's better to keep a kind of organization by folder, as it was by repo before merge :+1:

Reply:
Done. I also moved the interface & the `FileStorage` implementation.
In order to be consistent with `Message` I renamed the interface from `Storage` to `StorageInterface`

Reply:
The tree now looks like this:
![ - 31](https://user-images.githubusercontent.com/680206/78494514-bfa80900-7749-11ea-9084-10f84d75f09f.png)

The exception part is not consistent with the rest of the project but is it something we should worry about?

Reply:
LGTM